controllers/usersignup/usersignup_controller.go

```diff
@@ -357,7 +357,8 @@ func (r *Reconciler) checkIfMurAlreadyExists(reqLogger logr.Logger, config toolc
 
 			// if the Space is not Ready, another reconcile will occur when space status is updated since this controller watches space (without a predicate)
 			reqLogger.Info("Checking whether Space is Ready", "Space", space.Name)
-			if !condition.IsTrueWithReason(space.Status.Conditions, toolchainv1alpha1.ConditionReady, toolchainv1alpha1.SpaceProvisionedReason) {
+			if !(condition.IsTrueWithReason(space.Status.Conditions, toolchainv1alpha1.ConditionReady, toolchainv1alpha1.SpaceProvisionedReason) ||
+				condition.IsTrue(userSignup.Status.Conditions, toolchainv1alpha1.UserSignupComplete)) {
```

Comment:
just one minor comment about the form (not the content) - I find it easier to read using `AND` instead of `OR`, as @alexeykazakov wrote it in the previous comment:
```
if !condition1 && !condition2 {
  return error
}
``` 
but maybe it's just personal preference :)

Reply:
Ultimately it's the same logic, but I'll adjust it.

Fixed in ae1e668.